FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/IDicomDataClient.cs

```diff
@@ -12,12 +11,14 @@ namespace Microsoft.Health.Fhir.Synapse.DataClient
 {
     public interface IDicomDataClient
     {
-        public Task<List<string>> GetMetadataAsync(
-            ChangeFeedOptions changeFeedOptions,
-            CancellationToken cancellationToken = default);
-
-        public Task<long> GetLatestSequenceAsync(
-            bool includeMetadata,
+        /// <summary>
+        /// Returns server reponse content contains matching search results.
+        /// </summary>
+        /// <param name="dicomApiOptions">fhir api options.</param>
```

Comment:
Nit: `dicom api options` instead of `fhir`.

Reply:
Fixed, thanks.